typescript/src/ethereum.ts

```diff
@@ -39,11 +47,15 @@ export interface ContractConfig {
   /**
    * Address of the Ethereum contract as a 0x-prefixed hex string.
    */
-  address: string
+  address?: string
   /**
    * Signer that will sign all contract transactions.
    */
   signer: Signer
+  /**
+   * Number of a block in which the contract was deployed.
+   */
+  deployedAtBlockNumber?: number
```

Comment:
Let's make it clear in the docs what happens if the `deployedAtBlockNumber ` is not provided.